### PR TITLE
chore: Added the logo for the Label Bee Friendly

### LINKED ISSUE #7231: Add the logo for the Label Bee Friendly
The Label [Bee Friendly](https://fr.openfoodfacts.org/label/bee-friendly) is available on Open Food Facts, but the logo is missing. 

You can find the logo below in two different resolutions : 
- bee-friendly 90x90
- bee-friendly 1772x1772
 
Thank you :)

### PR DESCRIPTION
**Pull request by pratyaksh1610**
### Related issue(s) and discussion

- Fixes #7231

### CODE REVIEW

**Comment by stephanegigandet** [comment]
Could you rename the file to je-protege-les-abeilles.svg to je-protege-les-abeilles.90x90.svg and make the black background transparent?  90x90 indicates the aspect ratio.

**Comment by alexgarel** [comment]
See https://github.com/openfoodfacts/openfoodfacts-server/issues/7231#issuecomment-1251031479

You have to add the label to the taxonomy, otherwise it won't be displayed.

File: taxonomies/labels.txt [hunk 1]
```diff
@@ -19076,6 +19076,10 @@ pt:Maçãs da França
 origins:en: en:france
 ingredients:en: en:apple
 
+en:I protect the bees
+fr:Je protège les abeilles
+origins:en: en:france
```
Comment (alexgarel) [inline taxonomies/labels.txt:19081]: No the meaning of the label is not that honey comes from france

```suggestion
```

**Review by alexgarel — CHANGES_REQUESTED** [review]
thank @pratyaksh1610 
Still one fix to go :-)

**Comment by manon-corneille** [comment]
Hi guys, (cc @alexgarel)
It seems that the label Bee Friendly is not recognised yet → https://fr.openfoodfacts.org/label/bee-friendly 
Is it related to the " taxonomy rebuild "? 

Thanks :-)

**Pull request merged**